### PR TITLE
bpo-42153 doc: library imaplib a URL not available

### PR DESCRIPTION
**Pull request by Pixmew**
(https://docs.python.org/3/library/imaplib.html#imap4-objects) in Documentation of imaplib under (See Also :)
The link provided by University of Washington's IMAP Information Center is not Available (https://www.washington.edu/imap/)
As the link is taken down It should be replaced by another link.
It is replaced by the Link of Source Code provided by University of Washington's IMAP Information Center on Github for the same URL (https://github.com/uw-imap/imap) (Not Currently Maintained)

https://bugs.python.org/issue42153

### CODE REVIEW

File: Doc/library/imaplib.rst [hunk 1]
```diff
@@ -174,9 +174,9 @@ example of usage.
 
 .. seealso::
 
-   Documents describing the protocol, and sources and binaries  for servers
-   implementing it, can all be found at the University of Washington's *IMAP
-   Information Center* (https://www.washington.edu/imap/).
+   Source Code of Documents describing the protocol, sources and binaries for servers
+   implementing it, by the University of Washington's IMAP Information Center
+   can all be found at (https://github.com/uw-imap/imap) (**Not Maintained**).
```
Comment (birkenfeld) [inline Doc/library/imaplib.rst]: The parentheses around the URL are not necessary.

File: Doc/library/imaplib.rst [hunk 2]
```diff
@@ -174,9 +174,9 @@ example of usage.
 
 .. seealso::
 
-   Documents describing the protocol, and sources and binaries  for servers
-   implementing it, can all be found at the University of Washington's *IMAP
-   Information Center* (https://www.washington.edu/imap/).
+   Source Code of Documents describing the protocol, sources and binaries for servers
```
Comment (birkenfeld) [inline Doc/library/imaplib.rst]: "Source code of documents" makes no sense.  Further, I don't think binaries can be found in the source repo.

Comment (Pixmew) [reply Doc/library/imaplib.rst]: binaries are removed But what should i do of " Source Code of Documents describing"

**Comment by Pixmew** [comment]
edited as you suggested please check it

File: Misc/NEWS.d/next/Documentation/2020-11-15-13-46-31.bpo-42153.KjBhx3.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Source Code of Documents describing the protocol, sources and binaries for servers
+implementing it, by the University of Washington's IMAP Information Center
+can all be found at (https://github.com/uw-imap/imap) (**Not Maintained**).
```
Comment (gvanrossum) [inline Misc/NEWS.d/next/Documentation/2020-11-15-13-46-31.bpo-42153.KjBhx3.rst]: Usually small doc fixes like this don't need a news entry; if you really want a news entry, it should just read "Fix the URL for the IMAP protocol documents."

Comment (Pixmew) [reply Misc/NEWS.d/next/Documentation/2020-11-15-13-46-31.bpo-42153.KjBhx3.rst]: Isn't NEWS ENTRY required. The check fails when news entry is removed. Is there any way to skip news entry check for "Usually small doc fixes like this don't need a news entry" as you suggested?

**Review by gvanrossum — COMMENTED** [review]
I set a label "skip-news" so you don't need the news item.

**Comment by Pixmew** [comment]
Thank you Sir.
Really Looking forward to Contribute more.
one question :can I add or remove labels? just curious.

**Comment by gvanrossum** [comment]
For security reasons only people with triage or commit privilege can change labels.

**Pull request merged**